apps/workspace-host/src/interface.js

```diff
@@ -210,16 +231,24 @@ async
           } catch (err) {
             debug(`Couldn't find container: ${err}`);
           }
-        } else if (ws.state === 'running') {
-          if (!ws.launch_uuid) {
-            await workspaceUtils.updateWorkspaceState(ws.id, 'stopped', 'Shutting down');
-            await sqldb.queryAsync(sql.clear_workspace_on_shutdown, {
-              workspace_id: ws.id,
-              instance_id: workspace_server_settings.instance_id,
-            });
-          }
```

Comment:
@mwest1066 and I discussed this. As far as we can tell, it's impossible for a workspace to be in the `running` state and also have a null `launch_uuid`. Given that, we decided to remove this.